code/game/objects/items/weaponry.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const {Component} = require('bluespess');
+
+class BaseballBat extends Component {
+	constructor(atom, template) {
+		super(atom, template);
+	}
+}
+
+BaseballBat.depends = ["Item"];
+BaseballBat.loadBefore = ["Item"];
+
+BaseballBat.template = {
+	vars: {
+		components: {
+			"Item": {
+				inhand_lhand_icon: 'icons/mob/inhands/weapons/melee_lefthand.dmi',
+				inhand_rhand_icon: 'icons/mob/inhands/weapons/melee_righthand.dmi',
+				inhand_icon_state: "baseball_bat",
+				size: 4,
```

Comment:
tiny -> 1
small -> 2
normal-sized -> 3
bulky -> 4
**huge -> 5**
gigantic -> 6

Reply:
This should really be listed somewhere or converted into the JS equivalent of defines.